site/zenodo_rdm/templates/semantic-ui/profiler/index.html

```diff
@@ -0,0 +1,115 @@
+              <label for="base">Base profiler</label>
+            </div>
+          </div>
+          <div class="field">
+            <div class="ui checkbox">
+              <input type="checkbox" name="sql" {{ 'checked' if active_session.sql }}>
+              <label for="sql">SQL profiler</label>
+            </div>
+          </div>
+          <button class="ui positive button" type="submit">
+            <i class="play icon"></i>
+            Start
+          </button>
+        </form>
+        {% endif %}
+      </div>
+
+      {# Sessions list #}
+      <div class="nine wide column">
+        <h2>Sessions</h2>
+        {% if profiler_sessions %}
+        <form class="ui form" action="delete" method="post">
+          <button class="ui button" type="submit">
+            <i class="trash icon"></i>
+            Clear sessions
+          </button>
+        </form>
+        {% endif %}
+        <div class="ui accordion">
+          {% for session_id, reports in profiler_sessions.items() %}
+          <div class="title {{ 'active' if active_session.id == session_id }}">
+            <i class="dropdown icon"></i>{{ session_id }}
+          </div>
+          <div class="content {{ 'active' if active_session.id == session_id }}">
+            <div class="ui relaxed divided list">
+              {% for report in reports %}
+              {% set base_link %}
+                {%- if report.has_base_report -%}
+                <a target="_blank"
+                  href="{{ url_for('profiler.report_view', session_id=session_id, request_id=report.id, report_type='base') }}"
+                  class="header">Base</a>
+                {%- endif -%}
+              {% endset %}
+              {% set sql_link %}
+                {%- if report.has_sql_report -%}
+                <a target="_blank"
+                  href="{{ url_for('profiler.report_view', session_id=session_id, request_id=report.id, report_type='sql') }}"
+                  class="header">SQL</a>
+                {%- endif -%}
+              {% endset %}
+              <div class="item">
+                <i class="large file middle aligned icon"></i>
+                <div class="content">
+                  <div class="header">{{ report.context.method }} {{ report.context.url }}
+                    (<code>{{ report.context.endpoint }}</code>)</div>
+                  <div class="description">
+                    Referer: {{ report.context.referrer }}
+                    {{ base_link }} {{ sql_link }}
+                  </div>
+                </div>
+              </div>
+              {% endfor %}
+            </div>
+          </div>
+          {% else %}
+            <p>No recorded profiling sessions in storage yet.</p>
+          {% endfor %}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{% endblock page_body %}
+
+{% block javascript %}
+{{ super() }}
+<script type="text/javascript">
+  $('.ui.accordion').accordion();
```

Comment:
it was not working without it ? we might be missing js assets for semantic ui then... (not the react ones)

Reply:
Yeah, was also surprised that I had to call it manually... I just noticed we have [this guy](https://github.com/inveniosoftware/invenio-theme/blob/d1f59e10634f5f9ddb5450bed71b6aa685ec0381/invenio_theme/assets/semantic-ui/js/invenio_theme/theme.js#L45-L61), which might be changing a bit the behavior (i.e. what component toggles the accordion).